core/src/main/scala/akka/persistence/jdbc/query/dao/ByteArrayReadJournalDao.scala

```diff
@@ -60,6 +63,21 @@ trait BaseByteArrayReadJournalDao extends ReadJournalDao {
   }
 }
 
+object TagFilter {
+  /*
+   * Retains every event with tags that perfectly match passed tag.
+   * This is a workaround for bug https://github.com/akka/akka-persistence-jdbc/issues/168
+   *
+   * Builds a Source using the passed DatabasePublisher and
+   * filter out all events that where wrongly selected because of the usage ot LIKE operantor.
+   */
+  private[dao] def perfectlyMatchTag(
+      dbPublisher: DatabasePublisher[JournalRow],
+      tag: String): Source[JournalRow, NotUsed] = {
```

Comment:
doesn't matter, but this is conceptually a `Flow`, and could be defined as such

Reply:
I can refactor it. I don't have the reflex to think in terms of Flow. Will work on that. ;-)